src/main/java/com/facebook/openwifirrm/ucentral/UCentralConstants.java

```diff
@@ -8,6 +8,12 @@
 
 package com.facebook.openwifirrm.ucentral;
 
+/**
+ * Constants related to uCentral.
+ *
+ * @author rockymandayam
```

Comment:
remove this - we are not including `@author` tags generally (unless you copied the code from elsewhere)